free-3.md

```diff
@@ -10,41 +10,98 @@ _What need is met by your idea? Who are you helping? What is that person's
+rules in, to the different cards in, to actually having a computer play
+with a person would be interesting. I came up with it by reading Prof Ben's
+idea generation page. When he wrote down card games, I thought about my
+experience trying to learn Magic. I was not interested in thinking about
+the theory of the game because I was so new. However, it is such a 
+strategy based game that without understanding the theory and strategy of
+Magic, it's difficult to do well. I ended up playing with my brother,
+who was happy destroying me without a question, but if I had tried to
+play with a more experienced crowd I did not know, I would have felt like
+an outsider because of my lack of experience._
 
 ### Domain
 _Describe the project's domain in five words._
 
+_Teaching card game novices basics_
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+_The user would have to load their cards into the program and tell the language
+which version of the rules it ants to use. Programming looks like changing the
+cards and rules to be able to play a game against the computer. This is the
+right way to interact with the problem domain because there are so many cards
+for any given game that it would be ridiculous to include all of them.
+Additionally, card developers release new cards very often to continue to make
+a profit, so a designer could not account for every single card. However, each
+card has a certain template that it follows. So, once the language learns
+that template, it will be able to understand any card that the user programs
+into it._
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+_When a program runs, it will give the user a way to play the game with the
+computer. The errors that might occur would be incorrect cards. For instance,
+if a card did not have a certain field or the field was filled in with the
+wrong type, the program should not accept it as a card. So, it would tell
+the user which card was wrong and why and encourage the user to try the 
+program again with a fix. Additionally, if a user tried to make a move that
+was not allowed by the rules, the program should reset to the previous
+position and tell the user that the previous move was not allowed._
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+_It should be easy to make new cards for a given game. It shoud be difficult,
+but possible to make a completely new game because that would involve writing
+all of the rules into place and describing how a card is set up. It should be
+impossible to run a program in this language that just prints "Hello World."_
```

Comment:
I'm not entirely sure how, as I'm not familiar with Magic, but  [Magic is apparently Turing complete](http://www.toothycat.net/~hologram/Turing/). So you could do any program in Magic, although maybe not one that only prints "Hello World."